src/app/src/components/FilterSidebarFacilitiesTab.jsx

```diff
@@ -0,0 +1,328 @@
+import React, { Fragment } from 'react';
+import { arrayOf, bool, func, string } from 'prop-types';
+import { connect } from 'react-redux';
+import { Link } from 'react-router-dom';
+import CircularProgress from '@material-ui/core/CircularProgress';
+import Typography from '@material-ui/core/Typography';
+import Button from '@material-ui/core/Button';
+import List from '@material-ui/core/List';
+import ListItem from '@material-ui/core/ListItem';
+import ListItemText from '@material-ui/core/ListItemText';
+import Divider from '@material-ui/core/Divider';
+import get from 'lodash/get';
+import inRange from 'lodash/inRange';
+
+import ControlledTextInput from './ControlledTextInput';
+
+import {
+    makeSidebarSearchTabActive,
+    updateSidebarFacilitiesTabSearch,
+} from '../actions/ui';
+
+import { facilityCollectionPropType } from '../util/propTypes';
+
+import {
+    makeFacilityDetailLink,
+    downloadFacilitiesCSV,
+    getValueFromEvent,
+    caseInsensitiveIncludes,
+    sortFacilitiesAlphabeticallyByName,
+} from '../util/util';
+
+import COLOURS from '../util/COLOURS';
+
+import { filterSidebarStyles } from '../util/styles';
+
+const SEARCH_TERM_INPUT = 'SEARCH_TERM_INPUT';
+
+const makeFacilityListHeightPercentageString = () => {
```

Comment:
On my 1680x1050 screen there is some whitespace between the scrolling results list and the footer. If I shrink the browser window height the whitespace goes away. I looks like the conditional styling set up by this function may be the reason. Is there a way we can make sure that the scrolling area always fills the container?

<img width="643" alt="screen shot 2019-03-04 at 1 29 44 pm" src="https://user-images.githubusercontent.com/17363/53761437-ba132c00-3e82-11e9-8d84-f9b4b3032489.png">